crates/noirc_evaluator/src/ssa_refactor/ssa_builder/mod.rs

```diff
@@ -261,13 +293,17 @@ impl FunctionBuilder {
         self.insert_instruction(Instruction::ArraySet { array, index, value }, None).first()
     }
 
-    /// Terminates the current block with the given terminator instruction
+    /// Terminates the current block with the given terminator instruction.
+    ///
+    /// This is used to denote the block being completed, since basic blocks
+    /// can only have control flow instructions at the end of the block.
```

Comment:
"completed" seems a bit odd to me here. I can see what you were going for but it reads a bit to me like the block is finished and nothing can be inserted into it anymore, which isn't the case.